inc/cloner/class-cloner.php

```diff
@@ -1431,6 +1431,16 @@ protected function cloneSection( $section_id, $post_type, $parent_id = null ) {
 		// Remove items handled by cloneSectionMetadata()
 		unset( $section['meta']['pb_authors'], $section['meta']['pb_section_license'] );
 
+		if ( isset( $section['meta']['pb_part_invisible'] ) ) {
+			// pb_part_invisible metadata compatibility with previous boolean type (false | null)
+			if ( is_bool( $section['meta']['pb_part_invisible'] ) && $section['meta']['pb_part_invisible'] === false ) {
+				$section['meta']['pb_part_invisible'] = '';
+			}
+			if ( is_null( $section['meta']['pb_part_invisible'] ) ) {
+				$section['meta']['pb_part_invisible'] = 'on';
+			}
+		}
+
```

Comment:
for those source books with previous version containing `false` or `null` values, instead the new strings `""` or `"on"` just like others binary metadata values

Reply:
Nice detective work @richard015ar